src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.ts

```diff
@@ -341,15 +341,6 @@ export class InstructorCourseEnrollPageComponent implements OnInit {
     this.ngbModal.open(pasteModalBox);
   }
 
-  /**
-   * Reset page to default view
-   */
-  hideEnrollResults(): void {
-    this.showEnrollResults = false;
```

Comment:
So we can also remove the variable?

Reply:
Not really. This is only shown after enrollment, not all the time in the page.